picard/config.py

```diff
@@ -341,12 +319,51 @@ class ListOption(Option):
 setting = None
 persist = None
 
+PURGE_INTERVAL = 60000
```

Comment:
Since `PURGE_INTERVAL` is a number of milliseconds (according to https://doc.qt.io/qt-5/qtimer.html) it would be cleared to include unit in the name: `PURGE_INTERVAL_MILLISECONDS`.